src/routes/home/index.tsx

```diff
@@ -27,7 +27,9 @@ const Home = ({ name }: RouteChildProps) => {
   useDocumentTitle(name);
   const { t } = useTranslation();
   const { appState } = useAppState();
-  const { data } = useQuery<NodeData>(TOTAL_STAKED_QUERY);
+  const { data } = useQuery<NodeData>(TOTAL_STAKED_QUERY, {
```

Comment:
LGTM but I know @mattrussell36 has some ideas around refetching data when epochs end. Not sure if that would include home page data?

Reply:
Sorry my ticket wasn't really descriptive enough, but this wasn't what I was intending. What I'd like to do is add a component to query for epoch data and then refetch all queries if a new epoch has started. 

There are still some queries which we will have to intermittently poll but his should keep most of the data we get over the API as up to date as possible.